docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -409,7 +419,50 @@ load:
 
 .. code-block:: properties
 
-    exchange.base-directories=s3n://exchange-spooling-bucket-1,s3n://exchange-spooling-bucket-2
+    exchange.base-directories=s3://exchange-spooling-bucket-1,s3://exchange-spooling-bucket-2
+
+Google Cloud Storage
+~~~~~~~~~~~~~~~~~~~~
+
+If using Google Cloud Storage (GCS), it is recommended to configure a bucket lifecycle
+rule in GCS to automatically expire abandoned objects in the event of a node
+crash.
+
+GCS is mostly S3-compatible, with a few caveats. To enable exchange spooling on GCS in
+Trino, change the request endpoint to the Google Storage URI ``https://storage.googleapis.com``,
+and then configure your AWS access/secret keys to use the GCS HMAC keys. If you deploy
+Trino on GCP, you need to create a service account with access permission to your spooling
+bucket; otherwise, you need to configure the key path to your GCS credential file. For
```

Comment:
I do not get the `otherwise, you need to configure the key path to your GCS credential file` part. Don't we always need to do that?

Reply:
No, if you GCS deployment is associated with a service account that has access to the spooling bucket, then default authentication will just work out of the box.

Reply:
You mean GCP deployment, right? Makes sense.